tests/main/parallel-install-classic/task.yaml

```diff
@@ -25,7 +25,7 @@ prepare: |
 
     snap set system experimental.parallel-instances=true
 
-restore:
+restore: |
```

Comment:
I was wondering if we should add a static check that sounds the alarm if it hits: `restore:[ ]*$`, same for prepare most likely.